README.md

```diff
@@ -0,0 +1,163 @@
+another project that will read from AWS S3 bucket and will store all the data in SingleStore database. This
+will replace NEAR Indexer for Explorer PostgreSQL database at some moment and will become the main
+source for NEAR Explorer.
+
+## How to start
+
+The final setup consists of the following components:
+* AWS S3 Bucket as a storage
+* NEAR Lake binary that operates as a regular NEAR Protocol peer-to-peer node, so you will operate it as 
+  any other [Regular/RPC Node in NEAR](https://docs.near.org/docs/develop/node/rpc/hardware-rpc)
+
+### Prepare Development Environment
+
+Before you proceed, make sure you have the following software installed:
+* [Rust compiler](https://rustup.rs/) of the version that is mentioned in `rust-toolchain` file in the root of
+  [nearcore](https://github.com/nearprotocol/nearcore) project.
+* Ensure you have [AWS Credentials configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
+    From AWS Docs:
+
+  > For example, the files generated by the AWS CLI for a default profile configured with aws configure looks similar to the following.
+  >
+  > ~/.aws/credentials
+  > ```
+  > [default]
+  > aws_access_key_id=AKIAIOSFODNN7EXAMPLE
+  > aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+  > ```  
+  >  
+  > ~/.aws/config
+  > ```
+  > [default]
+  > region=us-west-2
+  > output=json
+  > ```  
+
+### Compile NEAR Lake
+
+```bash
+$ cargo build --release
+```
+
+### Configure NEAR Lake
+
+To connect NEAR Lake to the specific chain you need to have necessary configs, you can generate it as follows:
+
+```bash
+$ ./target/release/near-lake --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
+```
+
+The above code will download the official genesis config and generate necessary configs. You can replace `testnet` in the command above to different network ID (`betanet`, `mainnet`).
+
+**NB!** According to changes in `nearcore` config generation we don't fill all the necessary fields in the config file.
+While this issue is open https://github.com/nearprotocol/nearcore/issues/3156 you need to download config you want and replace the generated one manually.
+- [testnet config.json](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/testnet/config.json)
+- [betanet config.json](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/betanet/config.json)
+- [mainnet config.json](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/mainnet/config.json)
+
+Configs for the specified network are in the `--home-dir` provided folder. We need to ensure that NEAR Lake follows
+all the necessary shards, so `"tracked_shards"` parameters in `~/.near/testnet/config.json` needs to be configured properly.
+For example, with a single shared network, you just add the shard #0 to the list:
```

Comment:
neacore has a hard-coding that basically treats non-empty list of `tracked_shards` as "track all shards". Let's try to communicate this in this README as current statement is incorrect.

Reply:
If I'm not mistaken empty list means "do not track", and **any value** in the list means "track all shards"

Reply:
Correct

Reply:
ah, sorry I've misread your comment.